jupyterhub_config.py

```diff
@@ -35,11 +35,27 @@
 #  documentation for your spawner to verify!
 c.Spawner.cmd = ['cylc-uiserver']
 
-# --- Specify path to a logo image to override the Jupyter logo in the banner.
-
-c.JupyterHub.logo_file = str(DIST.joinpath(Path('img/logo.png')))
 
 # --- The class to use for spawning single-user servers.
 
 #  Should be a subclass of Spawner.
 c.JupyterHub.spawner_class = 'jupyterhub.spawner.LocalProcessSpawner'
+
+c.JupyterHub.ssl_cert = '/home/sanderso/cylchub.crt'
+c.JupyterHub.ssl_key = '/home/sanderso/cylchub.key'
```

Comment:
I think these two can be omitted?

Reply:
Whoops.

Review within 25 seconds, you even beat Codecov!